packages/sign_in_with_apple/lib/src/widgets/apple_logo_painter.dart

```diff
@@ -1,8 +1,7 @@
-/**
- * based on https://github.com/tomgilder/flutter_apple_sign_in/blob/8673cc2c0cf9067c3ab5dd392065df059e182b51/lib/apple_sign_in_button.dart
- * 
- * MIT License, Copyright (c) 2019 Rody Davis / https://github.com/tomgilder
- */
+/// based on https://github.com/tomgilder/flutter_apple_sign_in/blob/8673cc2c0cf9067c3ab5dd392065df059e182b51/lib/apple_sign_in_button.dart
+///
+/// MIT License, Copyright (c) 2019 Rody Davis / https://github.com/tomgilder
+///
```

Comment:
```suggestion
// based on https://github.com/tomgilder/flutter_apple_sign_in/blob/8673cc2c0cf9067c3ab5dd392065df059e182b51/lib/apple_sign_in_button.dart
//
// MIT License, Copyright (c) 2019 Rody Davis / https://github.com/tomgilder
//
```

3 / for doc comments.